opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -578,7 +577,7 @@ def __init__(
     ):
         # TODO: How should multiple TracerSources behave? Should they get their own contexts?
         # This could be done by adding `str(id(self))` to the slot name.
-        self._current_span_slot = Context.register_slot("current_span")
+        self.key = get_span_key(tracer_source_id=str(id(self)))
```

Comment:
Do we ever expect to have more than one `TracerSource`? I expected `TracerSource` to be a singleton, and all tracers to share the same current span .

Reply:
My takeaway from the SIG meeting this morning was that we expect a single TracerSource, so this change and the comment above could be removed if that's the case.